RELEASE.md

```diff
@@ -1,3 +1,46 @@
+# Release 1.12.0
+
+## Major Features and Improvements
+* tf.data users can now use `tf.data.experimental.AUTOTUNE` for the num_parallel_calls argument of `tf.data.Dataset.interleave`, `tf.data.Dataset.map`, and `tf.data.experimental.map_and_batch` to dynamically tune the level of parallelism at runtime.
+* Keras models can now be directly exported to the SavedModel format(`tf.contrib.saved_model.save_keras_model()`) and used with Tensorflow Serving.
+* TensorFlow builds with XLA support by default, does not enable XLA by default, but makes it accessible via default binary builds.
+
+## Bug Fixes and Other Changes
+
+* tf.data:
+  * tf.data users can now represent, get, and set options of TensorFlow input pipelines using `tf.data.Options()`, `tf.data.Dataset.options()`, and `tf.data.Dataset.with_options()` respectively.
+  * All C++ code moves to the `tensorflow::data` namespace.
+  * Adding support for `num_parallel_calls` to `tf.data.Dataset.interleave`.
```

Comment:
Nit: we should probably pick a tense and use it consistently throughout.  E.g. in this one list, we currently use "Adding", "Added" and "Add".  I'd just use "Add" everywhere, and change the other verbs to be consistent too.

Reply:
Done.